assets/aml-benchmark/components/src/aml_benchmark/batch_output_formatter/result_converters.py

```diff
@@ -103,6 +103,8 @@ def convert_result_ground_truth(self, result: Dict[str, Any]) -> Dict[str, Any]:
                 use_ground_truth_input = True
         elif self._model.is_aoai_model():
             use_ground_truth_input = True
+        elif self._model.is_vision_oss_model():
+            use_ground_truth_input = False
```

Comment:
I set it explicitly for readability and so that if we want to onboard vision models for batch quality pipeline in the future it is obvious where to make changes. I can revert if you want.